hikari/applications.py

```diff
@@ -137,15 +137,18 @@ class OAuth2Scope(str, enums.Enum):
     """
 
     APPLICATIONS_COMMANDS = "applications.commands"
-    """Allows your application's (slash) commands to be used in a guild.
+    """Allows your application's commands to be used in a guild.
 
     This is used in Discord's special Bot Authorization Flow like
     `OAuth2Scope.BOT` in-order to join an application into a guild as an
     application command providing integration.
     """
 
     APPLICATIONS_COMMANDS_UPDATE = "applications.commands.update"
-    """Allows your application to update it's (slash) commands via a bearer token."""
+    """Allows your application to update it commands via a bearer token."""
```

Comment:
```suggestion
    """Allows your application to update its commands via a bearer token."""
```

Reply:
I swear I only remove the `'`. What happened here